scripts/tests-functional/run.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
You may consider using a makefile in the future to help with some of the orchestration here. I've found they are really helpful for normal dev like 'CONFIG=kind make install' to deploy on your dev machine by creating a new kind cluster, etc. You can use HNC as an example:

https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/Makefile

Reply:
agreed, make would be a good option here